src/unittests/f32_interval.spec.ts

```diff
@@ -2630,6 +2631,41 @@ g.test('smoothStepInterval')
     );
   });
 
+interface PointToVectorCase {
+  input: number;
+  expected: IntervalBounds[];
+}
+
+g.test('unpack4x8unormInterval')
+  .paramsSubcasesOnly<PointToVectorCase>(
+    // prettier-ignore
+    [
+      // Some of these are hard coded, since the error intervals are difficult to express in a closed human readable
+      // form due to the inherited nature of the errors.
+      { input: 0x00000000, expected: [[hexToF32(0x81200000), hexToF32(0x01200000)], [hexToF32(0x81200000), hexToF32(0x01200000)], [hexToF32(0x81200000), hexToF32(0x01200000)], [hexToF32(0x81200000), hexToF32(0x01200000)]] },  // ~[0, 0, 0, 0]
```

Comment:
Might it be clearer to give these repeatedly used constants names?

Reply:
Done